src/main/webapp/web.xml

```diff
@@ -3,106 +3,12 @@
-        <servlet-name>deleteDriver</servlet-name>
-        <url-pattern>/drivers/delete</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>addDriver</servlet-name>
-        <servlet-class>taxi.controller.driver.AddDriverController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>addDriver</servlet-name>
-        <url-pattern>/drivers/add</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>allCars</servlet-name>
-        <servlet-class>taxi.controller.car.GetAllCarsController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>allCars</servlet-name>
-        <url-pattern>/cars</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>addCar</servlet-name>
-        <servlet-class>taxi.controller.car.AddCarController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>addCar</servlet-name>
-        <url-pattern>/cars/add</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>addDriverToCar</servlet-name>
-        <servlet-class>taxi.controller.car.AddDriverToCarController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>addDriverToCar</servlet-name>
-        <url-pattern>/cars/drivers/add</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>deleteCar</servlet-name>
-        <servlet-class>taxi.controller.car.DeleteCarController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>deleteCar</servlet-name>
-        <url-pattern>/cars/delete</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>allManufacturers</servlet-name>
-        <servlet-class>taxi.controller.manufacturer.GetAllManufacturersController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>allManufacturers</servlet-name>
-        <url-pattern>/manufacturers</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>addManufacturer</servlet-name>
-        <servlet-class>taxi.controller.manufacturer.AddManufacturerController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>addManufacturer</servlet-name>
-        <url-pattern>/manufacturers/add</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>deleteManufacturer</servlet-name>
-        <servlet-class>taxi.controller.manufacturer.DeleteManufacturerController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>deleteManufacturer</servlet-name>
-        <url-pattern>/manufacturers/delete</url-pattern>
-    </servlet-mapping>
+    <filter>
+        <filter-name>authenticationFilter</filter-name>
+        <filter-class>mate.web.filter.AuthenticationFilter</filter-class>
+    </filter>
+    <filter-mapping>
+        <filter-name>authenticationFilter</filter-name>
+        <url-pattern>/*</url-pattern>
+    </filter-mapping>
 </web-app>
```

Comment:
i am not sure that it is good practise to use web.xml and annotations together
but it is up to you and of course mentor)